packages/core/spec/Select-type.spec.ts

```diff
@@ -75,5 +76,13 @@ describe("Select<>", () => {
     >
   >;
 
+  type _fieldCallSelection = Select<
```

Comment:
maybe a test that checks that if you have a function call inside the selection of a function call?